website/templates/profile/oauth_app_detail.mako

```diff
@@ -96,7 +100,8 @@
     window.contextVars.urls = {
         webListUrl: ${ web_url_for('oauth_application_list') | sjson, n },
         apiListUrl: ${ app_list_url | sjson, n },
-        apiDetailUrl: ${ app_detail_url | sjson, n }
+        apiDetailUrl: ${ app_detail_url | sjson, n },
+        apiResetUrl: ${ app_reset_url | sjson, n },
```

Comment:
Actually, I think this variable is totally unused by the viewmodel, so this can be deleted entirely.